lib/index.js

```diff
@@ -83,8 +83,10 @@ GTM.prototype.page = function(page) {
 
 GTM.prototype.track = function(track) {
   var props = track.properties();
-  var id = this.analytics.user().id();
-  if (id) props.userId = id;
+  var userId = this.analytics.user().id();
+  var anonymousId = this.analytics.user().anonymousId();
+  if (userId) props.userId = userId;
+  if (anonymousId) props.anonymousId = anonymousId;
```

Comment:
feels like we should call this `segmentAnonymousId` or something namespaced like that. Whereas `userId` is customer-dictated, this is a value *we* create and populate. thoughts @f2prateek?